src/Apps/Artwork/Components/ArtworkSidebar2/ArtworkSidebar2Artists.tsx

```diff
@@ -0,0 +1,88 @@
+import { ShowMore, Text } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ArtworkSidebar2Artists_artwork } from "__generated__/ArtworkSidebar2Artists_artwork.graphql"
+import { RouterLink } from "System/Router/RouterLink"
+import styled from "styled-components"
+import { themeGet } from "@styled-system/theme-get"
+
+const ARTISTS_TO_DISPLAY = 4
+
+export interface ArtistsProps {
+  artwork: ArtworkSidebar2Artists_artwork
+}
+
+const StyledArtistLink = styled(RouterLink)`
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+  color: ${themeGet("colors.black100")};
+
+  &:hover {
+    text-decoration: underline;
+  }
+`
+
+// TODO: Tracking
+
+export const ArtworkSidebar2Artists: React.FC<ArtistsProps> = ({
+  artwork: { artists, culturalMaker },
+}) => {
+  if (!artists) return null
+
+  const showMoreText = `${artists.length - ARTISTS_TO_DISPLAY} more`
+
+  return (
+    <div>
```

Comment:
**question:** is there a reason that you picked a `<div>` here over a `<>` fragment or a `<Box>` ?

Reply:
Not really - it's a personal preference thing. Basically, why use `<>` when it's actually just a `<div>`? AFAIK there's not any advantage to it, it's just saving 3 characters (6 if you count the closing tag) and using components where we could be writing HTML sometimes gets us into trouble 🤷 

But I'm definitely not consistent about it either!

<img src="https://user-images.githubusercontent.com/5361806/187651101-364e86c2-fb27-48c4-82be-095db9637bb6.png" width="400px" />